src/features/auth/login/index.tsx

```diff
@@ -95,7 +107,9 @@ const Login = () => {
             onValidationChange={onValidationChange}
           />
 
-          <ForgotPasswordLink to={`${ROUTES.AUTH.ROOT}/${ROUTES.AUTH.FORGOT_PASSWORD}`}>
+          <ForgotPasswordLink
+            to={`${ROUTES.AUTH.ROOT}/${ROUTES.AUTH.FORGOT_PASSWORD}`}
+          >
```

Comment:
Some of the changes looks senseless like the one with ForgotPasswordLink, what is the purpose of them?
If there is no, please do not add them into the commits.
Generally, it is a good practice to separate new functionality commits from refactoring ones, it is much easier to do the review in this case.